test/Microsoft.ML.Benchmarks/PredictionEngineBench.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.IO;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Api;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class PredictionEngineBench
+    {
+        private PredictionEngine<IrisData, IrisPrediction> _irisModel;
+        private readonly string _irisModelPath = Program.GetInvariantCultureDataPath("iris_model.zip");
```

Comment:
>private readonly string _irisModelPath = Program.GetInvariantCultureDataPath("iris_model.zip"); [](start = 8, length = 95)

If I were to run *one* benchmark from this file, since these are field initializers *each* will run, which seems undesirable. This will be compounded when you stop loading from model files (which, as others have pointed out, you should not do) and instead just train a simple model from scratch. The benchmarks should be arranged in such a way that only those resources needed for that one benchmark actually get loaded. It might be fine if people run all of them *at once*, but if I'm focusing on something that might be less useful to me.